shell/initialize/App.js

```diff
@@ -30,6 +30,9 @@ export default {
   }),
 
   created() {
+    // Ensure we initialize i18n for all pages
+    this.$store.dispatch('i18n/init');
```

Comment:
If you have a better idea of where we should init this let me know. 

I moved this from the store/index because the store/index didn't actually call it for every page. Notably, if you refreshed the login page after changing the language it would switch back to english.

Reply:
Perhaps from within the install function of the i18n plugin itself? I think we have the store available to us at that point.

Honestly, I think that it fits here just as well. This is essentially the root of our app and dispatching some sort of init method makes sense at this point.

Reply:
I like the idea. It doesn't look like we currently expose the store to the install function and useStore is only evailable in the setup function within that plugin. Is there a reason we shouldn't expose it?

Reply:
We must install the plugin always after the store, since that's how it works, plus that's something which belongs to the plugin only and it's easier to find.
Since in the plugin install we have the options (as `_options`), should we not be able to dispatch it there?